packages/@react-spectrum/overlays/src/Popover.tsx

```diff
@@ -78,8 +81,9 @@ const PopoverWrapper = forwardRef((props: PopoverWrapperProps, ref: RefObject<HT
     maxHeight: null
   }, state);
 
+  // Attach Transition's nodeRef to outer most wrapper for node.reflow: https://github.com/reactjs/react-transition-group/blob/c89f807067b32eea6f68fd6c622190d88ced82e2/src/Transition.js#L231
```

Comment:
Why did you add the comment here and not to Modal.tsx or Tray.txs?

Reply:
No particular reason, I can definitely add that to those components as well